ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```diff
@@ -31,8 +31,8 @@ fi
 # that libcurl doesn't recognize the uppercase variables).
 if test -f "${SHARED_DIR}/proxy-conf.sh"
 then
-	# shellcheck source=/dev/null
```

Comment:
Other step changes would require additional approves, should these be moved to a different PR?

Reply:
I'm fine moving it to another PR, but this is a minor change, that brings consistency in shellcheck. If I struggle to get reviews then I'll remove the change in that file.